examples/oneapi/cpp/source/pca/pca_cor_dense_batch.cpp

```diff
@@ -14,30 +14,13 @@
 * limitations under the License.
 *******************************************************************************/
 
-#include <iomanip>
-#include <iostream>
-
 #include "oneapi/dal/data/table.hpp"
 #include "oneapi/dal/data/accessor.hpp"
 #include "oneapi/dal/algo/pca.hpp"
 
-using namespace oneapi;
-
-std::ostream &operator <<(std::ostream& stream, const dal::table& table) {
-    auto arr = dal::row_accessor<const float>(table).pull();
-    const auto x = arr.get_data();
+#include "service.hpp"
```

Comment:
According to our codestyle, we should use full paths.
```suggestion
#include "oneapi/cpp/source/utils/service.hpp"
```
Does it work?
@rlnx is this suggestion appropriate from your point of view?

Reply:
We added include utils folder in COPTS for make:
https://github.com/oneapi-src/oneDAL/blob/master/examples/oneapi/cpp/makefile_lnx#L112

Reply:
It should be CPATH pointing to the `sources` dir. It's better to stress on the fact utils included from examples. Use `util` instead `service`, it's  more aligned with library conventions:

```c++
#include "example_util/utils.hpp"
```

Reply:
Done

Reply:
example_**util**/**utils**.hpp sounds very excessively. Don't we want to rename the file? For example: 
```cpp
#include "example_util/ostream_helpers.hpp
```

Reply:
@makart19 , @amgrigoriev , @mchernov-intel  what do you think?

Reply:
Agree that for now it looks too excessive naming utils file with one output helper for table. But taking into consideration number of functions in current service.h for regular daal, i would suggest having several files, i.e. "output_helpers.hpp" (contains functions for printing tables, arrays ...), "input_helpers.hpp"(contains check_arguments, file_is_available...), "auxiliary.hpp" and utils.hpp which includes all of them.

Reply:
I'd rather don't create "utils.hpp" until this amount of auxiliary files becomes big enough

Reply:
Good suggestion, did just that.